src/assets/stylesheets/base/_font.scss

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: 'Neuzeit S LT Std Normal';
+  font-style: normal;
+  font-display: swap;
+
+  src: url('/assets/fonts/Neuzeit-S-LT-Std.ttf') format('truetype');
+}
+  
+@font-face {
+  font-family: 'Neuzeit S LT Std Bold';
+  font-style: normal;
```

Comment:
Should we set the `font-weight: bold` for this fone-face?

Reply:
Sure I've added `font-weight: bold ` to that style in https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28